src/docdb/getCosmosClient.ts

```diff
@@ -4,8 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { CosmosClient } from "@azure/cosmos";
+import * as https from "https";
+import * as vscode from 'vscode';
 import { appendExtensionUserAgent } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+
+export function getCosmosClient(endpoint: string, key: string, isEmulator: boolean | undefined): CosmosClient {
+
+    const vscodeStrictSSL: boolean | undefined = vscode.workspace.getConfiguration().get<boolean>(ext.settingsKeys.vsCode.proxyStrictSSL);
```

Comment:
Was this setting taken into account when creating a cosmos client in the old SDK?

Reply:
Yes, this was part of the last release with the old sdk. 
https://github.com/microsoft/vscode-cosmosdb/blob/9b4b3018fe34f1bccb9522b471bff6f1cccf1903/src/docdb/getDocumentClient.ts